src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java

```diff
@@ -77,9 +80,11 @@ public ImportEntriesDialog(BibDatabaseContext database, BackgroundTask<ParserRes
         Button btn = (Button) this.getDialogPane().lookupButton(importButton);
         btn.disableProperty().bind(booleanBind);
 
+        downloadLinkedOnlineFiles.setSelected(JabRefPreferences.getInstance().getFilePreferences().getDownloadLinkedFiles());
```

Comment:
If you like, you could also inject PreferencesService by the constructor. We are currently trying to get rid of the singleton pattern whereever possible.

Reply:
Sounds very reasonable, especially considered the service was already injected.
I resolved it in 50d2731